server/update_server.sh

```diff
@@ -16,4 +16,21 @@ for f in server/redis-migrations/*.js; do
   node -r "./babel-register" -r "core-js/proposals/reflect-metadata" -r "dotenv/config" "$f" || exit 1
 done
 
+echo "Syncing public assets to the cloud"
+ACCESS_KEY=$(echo $SB_FILE_STORE | jq '.doSpaces.accessKeyId')
+SECRET_KEY=$(echo $SB_FILE_STORE | jq '.doSpaces.secretAccessKey')
+HOST_BASE=$(echo $SB_FILE_STORE | jq '.doSpaces.endpoint')
+HOST_BUCKET="%(bucket)s.${HOST_BASE}"
+SPACE_NAME=$(echo $SB_FILE_STORE | jq '.doSpaces.bucket')
+
+s3cmd sync \
+  --access_key=${ACCESS_KEY} \
+  --secret_key=${SECRET_KEY} \
+  --host=${HOST_BASE} \
+  --host-bucket=${HOST_BUCKET} \
+  --recursive \
+  --delete-removed \
```

Comment:
I don't think deleting removed stuff is particularly good, ideally we want static assets to be available forever but we don't really want to have to keep them all in the repo forever